eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/CommonCodecs.scala

```diff
@@ -132,6 +132,7 @@ object CommonCodecs {
       .typecase(2, (ipv6address :: uint16).as[IPv6])
       .typecase(3, (base32(10) :: uint16).as[Tor2])
       .typecase(4, (base32(35) :: uint16).as[Tor3])
+      .typecase( 5, (variableSizeBytes(uint8, ascii) :: uint16).as[DnsHostname])
```

Comment:
nit:
```suggestion
      .typecase(5, (variableSizeBytes(uint8, ascii) :: uint16).as[DnsHostname])
```

Reply:
fixed in [e248b7c](https://github.com/ACINQ/eclair/pull/2234/commits/e248b7c043ffa4e74f61e3827264a9740ad612c4)